### DISCUSSION TITLE
web-top reference for documentation

### DISCUSSION THREAD

**Discussion by miohtama** [benoitc/gunicorn#2984] [Show and tell]
Based on scratching my own need, a built a UNIX top-like monitoring tool for gunicorn: [web-top](https://github.com/tradingstrategy-ai/web-top). It uses Gunicorn pre/post requests hooks and provides Text User Interface (TUI) live monitoring of Gunicorn servers.

I would like to include this in Instrumentation section in documentation if this is possible.

![image](https://user-images.githubusercontent.com/49922/186276030-d3bd44b1-c6dc-4868-b998-f0b7c613f266.png)

**Comment by benoitc** [comment]
@miohtama sure, please send a PR for it :)
